docker-compose.o11y.yml

```diff
@@ -0,0 +1,59 @@
+version: "3.9"
+
+services:
+  loki:

```

Comment:
the services (product, order,....) will send log and trace data to loki and tempo. Please help me verify that they can work fine even the loki, tempo is not up and running

Reply:
These exceptions will disappear after I run docker compose for observation.